src/Riok.Mapperly/Descriptors/DescriptorBuilder.cs

```diff
@@ -106,24 +108,59 @@ private void ReserveMethodNames()
         }
     }
 
-    private void ExtractObjectFactories()
+    private IEnumerable<IUserMapping> ExtractUserMappings()
```

Comment:
IMO it would be easier to just use `_mappings.UserMappings` in `EnqueueToBuildBody` instead yield return + ToList and passing as variable.

Reply:
I am not sure about this collection, it can be populated also from `ExtractExternalMappings` which should not be passed into `EnqueueUserMappings`.

This collection is populated after the `EnqueueUserMappings` call so it's not an issue right now. If you want to use it I can

Reply:
It's inside https://github.com/riok/mapperly/pull/681/commits/d931e58fb1758e0fb387ba307711037f8685ab40